draft-ietf-tls-tls13.md

```diff
@@ -1582,6 +1582,13 @@ when static RSA is used.
 Note: This is an update to TLS 1.2 so in practice many TLS 1.2 clients
 and servers will not behave as specified above.
 
+RFC EDITOR: PLEASE REMOVE THE FOLLOWING PARAGRAPH
```

Comment:
I'd noticed that, but the one in section 1 did the same thing, so I figured you all had decided that's what "following paragraph" meant. :-)

https://tlswg.github.io/tls13-spec/#rfc.section.1